tests/controllers/ServicesControllerTest.php

```diff
@@ -82,6 +81,17 @@ public function testServiceRequestWithFilters()
         $content = $this->decodeResponseJson();
         $this->assertEquals(0, count($content));
 
+        $this->doRequest('get', $this->apiUrl . '?uri=http://dev.foo/cultuurdienst');
+        $content = $this->decodeResponseJson();
+        $this->assertEquals(1, count($content));
```

Comment:
Check the expected uri:
`$this->assertEquals($content->uri, 'http://dev.foo/cultuurdienst');`